jenkins-pipelines/feature-ldap-authorization-and-audit-longevity-100gb-4h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-100gb-4h.yaml", "configurations/ldap-authorization.yaml", "configurations/audit.yaml"]''',
```

Comment:
I think we agreed that it can't work like that....
AFAIK the test config takes only one "configuration" parameter.
and AFAIR the test worked for you only when you added the audit config to the ldap config yaml.

Reply:
we thought it doesn't work, but it does work.
see this [job](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/ldap_with_audit/job/longevity-100gb-4h-ldap-authorization-with-audit/5):

```
14:22:21  Loading: /home/jenkins/slave/workspace/scylla-staging/fabio/ldap_with_audit/longevity-100gb-4h-ldap-authorization-with-audit/scylla-cluster-tests/defaults/test_default.yaml
14:22:21  Loading: /home/jenkins/slave/workspace/scylla-staging/fabio/ldap_with_audit/longevity-100gb-4h-ldap-authorization-with-audit/scylla-cluster-tests/defaults/aws_config.yaml
14:22:21  Loading: /home/jenkins/slave/workspace/scylla-staging/fabio/ldap_with_audit/longevity-100gb-4h-ldap-authorization-with-audit/scylla-cluster-tests/test-cases/longevity/longevity-100gb-4h.yaml
14:22:21  Loading: /home/jenkins/slave/workspace/scylla-staging/fabio/ldap_with_audit/longevity-100gb-4h-ldap-authorization-with-audit/scylla-cluster-tests/configurations/ldap-authorization.yaml
14:22:21  Loading: /home/jenkins/slave/workspace/scylla-staging/fabio/ldap_with_audit/longevity-100gb-4h-ldap-authorization-with-audit/scylla-cluster-tests/configurations/audit.yaml
```

also, once the environment started, see here the scylla.yaml:
```
scyllaadm@longevity-100gb-4h-auditing-db-node-f3818948-1:~$ grep ldap /etc/scylla/scylla.yaml
ldap_attr_role: cn
ldap_bind_dn: cn=admin,dc=scylla-qa,dc=com
ldap_bind_passwd: scylla
ldap_url_template: ldap://10.0.0.86:32770/dc=scylla-qa,dc=com?cn?sub?(uniqueMember=uid={USER},ou=Person,dc=scylla-qa,dc=com)
```
and the the audit keyspace created:
```
cassandra@cqlsh> DESCRIBE KEYSPACE audit 

CREATE KEYSPACE audit WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'}  AND durable_writes = true;

CREATE TABLE audit.audit_log (
    date timestamp,
    node inet,
    event_time timeuuid,
    category text,
    consistency text,
    error boolean,
    keyspace_name text,
    operation text,
    source inet,
    table_name text,
    username text,
    PRIMARY KEY ((date, node), event_time)
) WITH CLUSTERING ORDER BY (event_time ASC)
    AND bloom_filter_fp_chance = 0.01
    AND caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
    AND comment = ''
    AND compaction = {'class': 'IncrementalCompactionStrategy'}
    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
    AND crc_check_chance = 1.0
    AND dclocal_read_repair_chance = 0.0
    AND default_time_to_live = 0
    AND gc_grace_seconds = 864000
    AND max_index_interval = 2048
    AND memtable_flush_period_in_ms = 0
    AND min_index_interval = 128
    AND read_repair_chance = 0.0
    AND speculative_retry = '99.0PERCENTILE';
```

it means that we can have multiple config files, and no need to create a new one, or consolidate the 2...

Reply:
Ok, I guess.
Last time you said it doesn't work...

Reply:
> Ok, I guess.
> Last time you said it doesn't work...

yes, but it was an issue on my side, somewhere else :-)
now it is fixed